src/transformStateWithClones.js

```diff
@@ -6,8 +6,39 @@
  *
  * @return {Object[]}
  */
+function makeDeepCloneJSON(obj) {
+  return JSON.parse(JSON.stringify(obj));
+}
+
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyOfState = makeDeepCloneJSON(state);
```

Comment:
let's keep it simple
```suggestion
  const stateCopy = { ...state };
```